package-lock.json

```diff
@@ -1,8576 +0,0 @@
-{
```

Comment:
deleting this file is wrong

you're introducing dep changes

here's how it works:

1. if you're not altering deps, don't alter package lock at all
2. if you're altering deps, delete node_modules, delete package-lock, then generating a new one with `npm i` and commit that

do the latter

also you need to account for [express variables](https://github.com/rooseveltframework/roosevelt#express-variables-exposed-by-roosevelt)